tools/olm-bundle/generate-olm-bundle.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+#
+# Generates OLM bundle using existing CRDs
+#
+#
+PACKAGE_NAME=strimzi-cluster-operator
+VERSION=$1
+CHANNELS=latest
+PREVIOUS_BUNDLE_VERSION=$(curl -s https://raw.githubusercontent.com/operator-framework/community-operators/master/community-operators/strimzi-kafka-operator/strimzi-kafka-operator.package.yaml | yq e '.channels[0].currentCSV' -)
+
+CSV_TEMPLATE_DIR=./csv-template
+CSV_TEMPLATE=${CSV_TEMPLATE_DIR}/bases/${PACKAGE_NAME}.clusterserviceversion.yaml
+
+CRD_DIR=../../packaging/install/cluster-operator
+MANIFESTS=./bundle/manifests
+CSV_FILE=${MANIFESTS}/${PACKAGE_NAME}.clusterserviceversion.yaml
+
+if [ -z "$1" ]; 
+then
+    echo "No <BUNDLE_VERSION> argument supplied"
+    echo "Try running:"
+    echo "   ./generate-olm-bundle.sh <BUNDLE_VERSION>"
+    echo "e.g:"
+    echo "   ./generate-olm-bundle.sh 2.3.0"
+    exit 1
+fi
+
+# Generates bundle from existing CRDs
+generate_olm_bundle() {
+  # To generate a bundle from existing CRDs using the operator-sdk we must:
+  #   1.) Specify package name "--package" and bundle version "--version"
+  #   2.) Set `--input-dir` to the directory containing CRDs
+  #   3.) Place our template CSV file in a directory in the kustomize format: 
+  #       "<some-dir>/bases/<package-name>.clusterserviceversion.yaml"
+  # *Note* the package name must match the name used in the CSV template
```

Comment:
What will happen if it doesn't? If it would result in subtle bugs could we do a check here as we know `$PACKAGE_NAME` and can use yq to inspect the CSV?